docs/snippets/common/arg-types-in-story.ts.mdx

```diff
@@ -0,0 +1,26 @@
+```ts
+// Button.stories.ts|tsx
+
+// Replace your-renderer with the renderer you are using (e.g., react, vue3, angular, etc.)
+import type { Meta, StoryObj } from '@storybook/your-renderer';
+
+import { Button } from './Button';
+
+const meta: Meta<typeof Button> = {
+  component: Button,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Button>;
+
+export const Basic: Story = {
+  argTypes: {
+    // 👇 This story expects a label arg
+    label: {
+      control: 'string',
+      description: 'Overwritten description',
+    },
```

Comment:
is this the same as 
```ts
label: {
      control: {type:'text'},
      description: 'Overwritten description',
    },
```

Reply:
Not the same. `control: 'string'` isn't valid! 😅  Thanks for the catch! 🙏